src/server/utils/processDocInfos.ts

```diff
@@ -25,6 +26,19 @@ export function processDocInfos(
         );
       }
       const route = url.substr(baseUrl.length);
+
+      // ignore files
+      if (
+        ignoreFiles?.some((reg: RegExp | string) => {
+          if (typeof reg === "string") {
+            return route === reg;
+          }
+          return route.match(reg);
+        })
+      ) {
+        return;
+      }
```

Comment:
@wfatec Test coverage decreased, and these lines above are not covered. See https://coveralls.io/builds/40073221/source?filename=src%2Fserver%2Futils%2FprocessDocInfos.ts#L33